lib/ffmpeg/encoderbase.py

```diff
@@ -141,6 +141,16 @@ def before(self):
     self.refctx = []
     self.post_validate = lambda: None
 
+  def get_lowpower_opt(self):
+    if get_media()._get_gpu_gen() < 13:
+         #only cover "low_power=0" path
+         return 0
+    else:
+         #For MTL platforms (gen >== 13)
+         #will add random() to set "low_power=0" and no "low_power" option
+         #to cover more test path in future
+         return 0
+
```

Comment:
I don't like this solution.  It's confusing.  Why can't we just let MTL+ use the normal lowpower=0 in the tests to start.

Reply:
ok, removed